shark/iree_utils/compile_utils.py

```diff
@@ -358,6 +358,10 @@ def load_vmfb_using_mmap(
     flatbuffer_blob_or_path, device: str, device_idx: int = None
 ):
     print(f"Loading module {flatbuffer_blob_or_path}...")
+    if "vulkan" in device:
+        # Vulkan pipeline creation consumes significant amount of time.
+        print("(This step may take a few minutes)")
```

Comment:
nit: You could specify `compiling shaders` or something similar.

Reply:
Thanks. I added this and moved the message closer to where the compilations happen.